.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release
+on:
+  push:
+    branches:
+      - 'release/*'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup Java JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+      - name: Build project
+        run: ./gradlew assemble
```

Comment:
This only do assemble...
Don't we need release plugin replace SNAHOP version to empty string?

Reply:
`assemble` doesn't run a test task. How about replacing it with `build` task?

Reply:
@junoyoon 
This action is just create release & tag when pushed `release/*` branch.
So if we want to replace SNAPSHOT version to empty string, we'll have to do it by self.

I was some confusion because the operation of the `gradle-release plugin` was different from the current release method.
It seems to have a similar operation to what you said this https://github.com/naver/scavenger/issues/52#issuecomment-1485345568, so I'll check again

Reply:
@junoyoon 

We can use `gradle-release` plugin **if merging `release` branch to `main` is ok.**
Using `gradle-release` commits tag version and new version at working branch. 
We want to create tag at release branch, we can use `pushReleaseVersionBranch ` option. ([reference](https://github.com/researchgate/gradle-release#introduction))
But using `pushReleaseVersionBranch` will checkout to the release branch and merge from the working branch.
If we don't use `pushReleaseVersionBarnch` option, just commit working branch.(in our case maybe working branch is `main`)

If we decide to use `gradle-release` plugin, I'd like to proceed release step as follows.
```
1. Checkout release branch and commit release version by developer.
  - It can be replaced with the gradle release plugin, but for some reason it is difficult to control it with the github action.
2. Using github actions on push event at release branch, trigger gradle-release plugin. 
  - This step is creating TAG.
3. Using github actions on tag event, create Release.
```

cc. @naver/scavenger-dev

Reply:
i will review this weekend? ok?

Reply:
@junoyoon 
https://github.com/naver/scavenger/pull/55#discussion_r1151478029 is outdated.
plz refer https://github.com/naver/scavenger/pull/55#issuecomment-1489993218
When release, we only have to Pull Request `develop` to `main` branch.
Other tasks are done as Github Actions.